src/main/java/nl/mpdev/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -3,76 +3,67 @@
 import nl.mpdev.backend_spring_boot_tech_it_easy_controller.exceptions.RecordNotFoundException;
 import nl.mpdev.backend_spring_boot_tech_it_easy_controller.exceptions.StringTooLongException;
 import nl.mpdev.backend_spring_boot_tech_it_easy_controller.models.Television;
+import nl.mpdev.backend_spring_boot_tech_it_easy_controller.services.TelevisionService;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/api/v1")

```

Comment:
Hier mag je nog "/televisions" aan toevoegen, aangezien je dat in elk van je mappings gebruikt.